src/message_interface/client_method.rs

```diff
@@ -467,4 +472,40 @@ pub enum ClientMethod {
         /// Block
         block: BlockDto,
     },
+    /// Returns a transaction ID
```

Comment:
Maybe also add that it's the hash of the provided transaction payload

Reply:
fixed